overlord/ifacestate/ifacemgr.go

```diff
@@ -178,6 +178,19 @@ type ConnectionState struct {
 	// Undesired indicates whether the connection, otherwise established
 	// automatically, was explicitly disconnected
 	Undesired bool
+	PlugAttrs map[string]interface{}
+	SlotAttrs map[string]interface{}
+}
+
+func mergeAttrs(one map[string]interface{}, other map[string]interface{}) map[string]interface{} {
```

Comment:
Could you please add a comment to the method stating that the assumption is the maps are disjoinit (no common keys), and the copy is shallow?